pub-sub/README.md

```diff
@@ -88,15 +88,15 @@ The Dapr CLI provides a mechanism to publish messages for testing purposes. Now,
    
     * Linux/Mac:
         ```bash
-        dapr publish --topic A --data '{ "message": "This is a test" }'
+        dapr publish --topic A --pubsub pubsub --data '{ "message": "This is a test" }'
```

Comment:
it would need change in https://github.com/dapr/quickstarts/blob/master/pub-sub/deploy/redis.yaml#L4 for k8s. Lets change the name to be pubsub

Reply:
Right, changed it